### ISSUE TITLE
As a user I am able to see services supported by a given provider by clicking on a provider's name link in a service detail description

### ISSUE THREAD

**Issue by roksanaer** [cyfronet-fid/marketplace#111]
After clicking on provider's name given in a service detail (https://v81k42.axshare.com/#g=1&p=service-detail_v3 - Onemikro) user is redirected to the provider's service list- Provider's service list should be analogical to the category/filtered service list (https://v81k42.axshare.com/#g=1&p=services-list_v3). @abacz add any additional info if necessary

**Comment by roksanaer** [comment]
connected to #110

**Comment by abacz** [comment]
When you click to oneMikro provider in first item you will see provider services.
https://v81k42.axshare.com/#g=1&p=services-list_provider

How do you think about special header for this kind of list with provider name? I' m thinking about it beacouse it will be unique situation where filter has own header.

**Comment by roksanaer** [comment]
all in for a dedicated header

**Comment by roksanaer** [comment]
IMPORTANT comment:
service provider is an entity supporting the delivery of a service; institution offering a given service; one service might have many providers, a provider can have many services, thus relation in the DB should be many to many

additionally, service owner is not equal to the service provider: a service owner is a person responsible for the service entry (real person with a unique ID in the aai) who has all provilages to edit the service entry in the MP. service provider (at least at the moment) should be an entity in the MP which is an element of a data model for the services. each service should have an attached service provider(s) but at the moment it is only a way of a categorization in a way. there is no real person or organization standing behind it, who can manage services within the service provider entity. 

our goal is to enable the management of the service offers in a scope of a service provider but it is our goal for the future

to sum up:
service provider - relation many to many with a service; entity offering a service but an entity which is not manageable in the beginning - existing only in our data model; a way of organizing the services
service owner - relation one to many (a service owner can have many services) - MP user, assigned to a service responsible for a service entry

**Comment by martaswiatkowska** [comment]
Sooory that it is not ended.
All that I have been done I pushed to brunch 111-service-provider-view
Firstly I implemented that exactly like categories. 
It is needed to implement the sending parameters from check-boxes to index in controller and then filter the services ( that's what I wanted implemented )
